### PR TITLE
chore: trial changes

### PR DESCRIPTION
**Pull request by ArmanNik**

### CODE REVIEW

File: src/lib/system.ts [hunk 1]
```diff
@@ -24,3 +24,4 @@ export const isCloud = MODE === Mode.CLOUD;
 export const isSelfHosted = MODE !== Mode.CLOUD;
 export const hasStripePublicKey = !!VARS.STRIPE_PUBLIC_KEY;
 export const GRACE_PERIOD_OVERRIDE = false;
+export const TRIAL_PERIOD_OVERRIDE = true;
```
Comment (stnguyen90) [inline src/lib/system.ts:27]: This is an override to disable trials? It's a little confusing..I thought it should have been false to disable trials 🧐 . Would it be clearer to name it `DISABLE_TRIAL_PERIOD`?

**Review by stnguyen90 — APPROVED** [review]

File: src/lib/system.ts (hunk 1 above)
Comment (stnguyen90) [reply src/lib/system.ts:27]: If anything, maybe add a comment? Anyways, you're free to merge whenever.

File: src/routes/console/wizard/cloudOrganization/confirmDetails.svelte [hunk 2]
```diff
@@ -123,9 +126,9 @@
         {#if $createOrganization.billingPlan !== BillingPlan.STARTER}
             <p class="text u-margin-block-start-16">
                 This amount, and any additional usage fees, will be charged on a recurring 30-day
-                billing cycle after your trial period ends on <b
-                    >{toLocaleDate(billingPayDate.toString())}</b
-                >.
+                billing cycle{TRIAL_PERIOD_OVERRIDE
+                    ? '.'
+                    : ` after your trial period ends on ${toLocaleDate(billingPayDate.toString())}`}.
```
Comment (stnguyen90) [inline src/routes/console/wizard/cloudOrganization/confirmDetails.svelte:131]: There seems to be an extra period:

image

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Review by lohanidamodar — APPROVED** [review]
Looks good.

**Pull request merged**